src/styles/date.scss

```diff
@@ -0,0 +1,8 @@
+.ob-input[type='date']::-webkit-calendar-picker-indicator,
+.ob-input[type='time']::-webkit-calendar-picker-indicator,
+.ob-input[type='datetime-local']::-webkit-calendar-picker-indicator {
+  right: 0;
+  position: absolute;
+  padding: 11px;
+  opacity: 0;
+}
```

Comment:
This seems browser specific. Is this going to work as expected on all browsers?

Reply:
Works so far on safari and chrome.

Reply:
And it's how all the cool bloggers do it